company_linkage/sequences/initial_data.csv

```diff
@@ -1,2 +1,2 @@
 high_resolution_entities,organizations
-high_resolution_entities,original_company_names
```

Comment:
This is why the updates to company names weren't getting picked up, they were being written to the wrong dataset